### DISCUSSION TITLE
Cancel mail

### DISCUSSION THREAD

**Discussion by MichaelWybo** [liedekef/events-made-easy#195] [General]
Hello Franky is there a way to differentiate the email sent when the user cancels a registration / reservation and the email sent when the organizer cancels an event ?

**Comment by liedekef** [comment]
Well, it seems that I currently send out the cancelBooking-related mails when an event is trashed, while I probably should send the trashBooking-related mails (the ones being sent when an admin moves a booking to trash).
However, cancelBooking is more correct as the booking is not moved to trash but really deleted ...
Maybe: try using [eme_if] in the cancelBooking emails and check for the condition #_IS_ADMIN , then you can change the text conditionally inside the mail itself.

**Comment by MichaelWybo** [comment]
Will try these nice workaround.

PS: #_IS_ADMIN  is not documented :)

**Reply by liedekef** [reply]
Indeed, it is in fact an alias for #_IS_ADMIN_PAGE. I corrected the doc.
